src/assessment.handlebars

```diff
@@ -1043,8 +1043,9 @@
   </div>
   <div class="slide main-menu" id="main-menu">
     <div class="flow">
-      <div class="center donation-banner">Text SEAP02 £2 to 70070 to help us support more people like you.
-          <img src="images/jtg_logo.png" alt="Just Text Giving logo"/>
+      <div class="donation-banner">
+        <p class="center">If you find this website useful, please consider donating.</p>
+        <p class="center">Text <b>'SEAP02 £2'</b> to <b>70070</b> to help us support more people like you.</p>
```

Comment:
Same as on esa - judgement of whether to use the html entity or leave it as it is. Not sure if the encoding is necessary anymore  - but may have been an accessibility enhancement - but would have to look into it.

Interesting: https://www.deque.com/blog/dont-screen-readers-read-whats-screen-part-1-punctuation-typographic-symbols/

Reply:
https://www.levelaccess.com/csscontentproperty/

![screenshot_20181109_120553](https://user-images.githubusercontent.com/861172/48261720-e0b2e000-e417-11e8-84da-68ef8cf3ff03.png)